src/main/java/gr/cytech/chatreminderbot/rest/controlCases/Control.java

```diff
@@ -171,6 +180,11 @@ public String caseShowReminders() {
         return caseShowReminders.showReminders();
     }
 
+    public String caseShowSettings(){
+        caseShowsettings.setRequest(request);
+        return caseShowsettings.showSettings();
```

Comment:
perhaps you can pass the request as an argument of `showSettings` straight away and skip the whole instance field setting altogether?